x-pack/plugin/data-streams/src/test/java/org/elasticsearch/xpack/datastreams/mapper/DataStreamTimestampFieldMapperTests.java

```diff
@@ -277,17 +277,17 @@ public void testValidateNotDisallowedAttribute() throws IOException {
         assertThat(e.getMessage(), equalTo("data stream timestamp field [@timestamp] has disallowed attributes: [store]"));
     }
 
-    public void testCannotUpdateTimestampField() throws IOException {
+    public void testCanUpdateTimestampField() throws IOException {
```

Comment:
I think we need to adjust test and the data stream timestamp field mapper.
Updating from `false` to `true` is allowed, but updating from `true` to `false` is disallowed.
`

Reply:
This validation can be added by invoking `setMergeValidator(...)`method when building enabled parameter in `DataStreamTimestampFieldMapper` line 78.